fedora-spec-files/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM fedora:35
+
+# `install_weak_deps=False` so we don't get texlive and other deps which we don't need.
+RUN dnf install -y \
+    python3-notebook \
+    python3-jupyter-client \
+    python3-matplotlib \
+    python3-bodhi-client \
+    python3-koji \
+    python3-tabulate \
+    xz \
+    --setopt=install_weak_deps=False
+
+RUN cd / \
+    && curl -sO https://src.fedoraproject.org/repo/rpm-specs-latest.tar.xz \
+    && tar -xf ./rpm-specs-latest.tar.xz
```

Comment:
For more flexibility in working with the data, I would make this a `download-data` target in the Makefile, something like:

```
rpm-specs: rpm-specs-latest.tar.xz
...

unpack-archive: rpm-specs
...

download-data: unpack-archive
...

generate-data: download-data
...
```

Reply:
oh, we can actually use make the way it's meant to be used, take care of files and their dependencies, this is a great suggestion